Utilities/constants.py

```diff
@@ -0,0 +1,34 @@
+"""
+All of the constant values used across all of the scripts are located here.
+"""
+
+# Safety
+DEFAULT_VELOCITY = 0.50
+VELOCITY_THRESHOLD = 2  # never let the drone go faster than 1 m/s for safety (is this a good number?)
```

Comment:
Velocity is a vector, not a scalar value. You're thinking of speed here.


```suggestion
SPEED_THRESHOLD = 2  # never let the drone go faster than 1 m/s for safety (is this a good number?)
```